python/pyspark/ml/wrapper.py

```diff
@@ -223,3 +223,20 @@ def _call_java(self, name, *args):
         sc = SparkContext._active_spark_context
         java_args = [_py2java(sc, arg) for arg in args]
         return _java2py(sc, m(*java_args))
+
+
+class JavaCallable(object):
```

Comment:
Ideally, I would have liked to reuse parts from `JavaWrapper` and `JavaModel` but just making this small class was the most simple solution and least intrusive.